VolleyManagement.UI/Areas/WebAPI/Controllers/TournamentsController.cs

```diff
@@ -107,20 +107,14 @@ public List<PivotStandingsViewModel> GetTournamentPivotStandings(int id)
         [Route("api/Tournament/{tournamentId}/Schedule")]
         public ScheduleViewModel GetSchedule(int tournamentId)
         {
-            List<GameViewModel> gamesViewModel = _gameService.GetTournamentGames(tournamentId)
-                                                        .Select(t => GameViewModel.Map(t)).ToList();
-            foreach (var item in gamesViewModel)
-            {
-                if (item.Result.TotalScore.IsEmpty)
-                {
-                    item.Result = null;
-                }
-            }
+            var games = _gameService.GetTournamentGames(tournamentId)
```

Comment:
It seems for me we don't need invoke ToList()

Reply:
I'll fix that